x-pack/plugins/ml/server/routes/system.ts

```diff
@@ -150,7 +156,11 @@ export function systemRoutes(
     {
       path: '/api/ml/ml_node_count',
       validate: false,
+      options: {
+        tags: ['access:ml:canGetJobs'],
```

Comment:
Just curious as to why this isn't `canAccessML` like the other routes in here?

Reply:
`canAccessML` means it's also available on basic.
An end user shouldn't need to query the ml node count if they only have a basic license.